store/sqlstore/upgrade.go

```diff
@@ -551,12 +550,8 @@ func upgradeDatabaseToVersion50(sqlStore *SqlStore) {
 	//    DELETE from Systems WHERE Name = 'migration_advanced_permissions_phase_2';
 
```

Comment:
I can see a line `sqlStore.CreateColumnIfNotExists("TeamMembers", "DeleteAt", "bigint(20)", "bigint", "0")` in `upgradeDatabaseToVersion32`.

But in our case, the `DeleteAt` column is part of the create table command. Wouldn't this fail for installations upgrading from 3.1 to 5.34 for example?

My understanding was that we would run _all_ migrations for Teams/TeamMembers from version 3.0. But it looks like some are missing.

Reply:
Totally, i've missed that! 🦅 👀